sematic/db/migrations/20221208132055_add_run_clone_id.sql

```diff
@@ -0,0 +1,6 @@
+-- migrate:up
+ALTER TABLE runs ADD COLUMN cloned_from_run_id character(32);
```

Comment:
How about `original_run_id` to make it shorter?

Reply:
Done.